src/test/java/seedu/address/logic/parser/PeoplePeopleAddCommandParserTest.java

```diff
@@ -31,7 +31,8 @@
 
```

Comment:
The file name looks like it has a extra People HAHA and same for the below People command parser tests

Reply:
Would also be good to group the People Parser Tests into the parser.people package as well, same idea as for the command tests